django/cantusdb_project/main_app/templates/chant_detail.html

```diff
@@ -129,14 +129,17 @@ <h3>{{ chant.incipit }}</h3>
                         </div>
                     {% endif %}
 
-                    {% if chant.differentiae_database %}
+                    {% if chant.diff_db %}
                         <div class="col">
                             <dt>Differentia&nbsp;Database</dt>
                             <dd>
-                                <a href="https://differentiaedatabase.ca/differentia/{{ chant.differentiae_database }}" target="_blank">
-                                    {{ chant.differentiae_database }}
+                                {% if chant.diff_db.melodic_transcription %}
```

Comment:
if a chant's `diff_db` is `None`, do we not get an error message when the template tries to display `None.melodic_transcription`? Not certain of this, but do make sure this works for a chant with no `diff_db`.

Reply:
If `diff_db` is None this line will not be evaluated since line 132 already checks for this situation I believe